components/AppNav.tsx

```diff
@@ -39,63 +32,48 @@ const navItems: NavItem[] = [
   }
 ]
 
-const NavBar: React.FC<AuthLinkProps> = ({ session }) => {
+const NavBar: React.FC<AuthLinkProps> = ({}) => {
   const router = useRouter()
-  const isAdmin = _.get(session, 'user.isAdmin', false) as boolean
   const location = router.asPath
+
   return (
     <>
       {navItems.map(button => (
         <NavLink
           {...button}
-          className={`${styles['nav-item']} nav-link`}
+          className={`nav-link ${styles['nav-item']} ${styles['nav-link']} px-3`}
           key={button.name}
           activePath={location === button.path}
         >
           {button.name}
         </NavLink>
       ))}
-      {isAdmin && (
-        <DropdownMenu
-          title="Admin"
-          items={dropdownMenuItems}
-          bsPrefix={styles['dropdown-item']}
-        />
-      )}
     </>
   )
 }
 
-const LoggedInAuthNav: React.FC<{ username: string }> = ({ username }) => {
-  const capitalized = username.charAt(0).toUpperCase() + username.slice(1)
+const LoggedInAuthNav: React.FC<{ username: string; session: any }> = ({
```

Comment:
Please don't use `any` as a type definition when possible. It defeats the purpose of using typescript